libraries/chain/smt_evaluator.cpp

```diff
@@ -162,10 +164,18 @@ void smt_setup_evaluator::do_apply( const smt_setup_operation& o )
       token_ico_obj.launch_time = o.launch_time;
       token_ico_obj.steem_units_soft_cap = o.steem_units_soft_cap;
       token_ico_obj.steem_units_hard_cap = o.steem_units_hard_cap;
+      token_ico_obj.steem_units_min = o.steem_units_min;
    } );
 
    smt_setup_evaluator_visitor visitor( token_ico, _db );
    o.initial_generation_policy.visit( visitor );
+
+   if ( !_db.is_pending_tx() )
```

Comment:
This is going to be a required check for any action that is pushed during as a result of a transaction. Let's move this check in to `push_required_action`.